dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractHostedEntryResource.java

```diff
@@ -265,11 +271,12 @@ protected String createValidationMessages(U version) {
      * Note: There is one validation entry for each sourcefile type. This is true for test parameter files too.
      * @param version Version to validate
      * @param entry Entry for the version
+     * @param mainDescriptor
```

Comment:
Javadoc for why this method might need  a mainDescriptor now

Reply:
Fixed in upcoming commit